src-tauri/src/systemtray_manager.rs

```diff
@@ -101,19 +107,23 @@ impl SystemtrayManager {
         match current_os {
             CurrentOperatingSystem::Windows => {
                 format!(
-                    "Hashrate | Usage\nCPU: {:.0} H/s | {:.0}%\nGPU: {:.0} H/s | {:.0}%\nEst. earning: {} tXTM/day",
-                    data.cpu_hashrate,
+                    "Hashrate | Usage\nCPU: {} H/s | {:.0}%\nGPU: {} H/s | {:.0}%\nEst. earning: {} tXTM/day",
+                    Formatter::new().with_decimals(2).with_separator("").format(data.cpu_hashrate),
                     data.cpu_usage,
-                    data.gpu_hashrate,
+                    Formatter::new().with_decimals(2).with_separator("").format(data.gpu_hashrate),
                     data.gpu_usage,
-                    format_balance(data.estimated_earning)
+                    Formatter::new().with_decimals(2).with_separator("").format(data.estimated_earning / 1_000_000.0)
                 )
             }
             CurrentOperatingSystem::Linux => "Not supported".to_string(),
             CurrentOperatingSystem::MacOS => {
                 format!(
-                    "CPU:\n  Hashrate: {:.0} H/s\n  Usage: {:.0}%\nGPU:\n  Hashrate: {:.0} H/s\n  Usage: {:.0}%\nEst. earning: {} tXTM/day",
-                    data.cpu_hashrate, data.cpu_usage, data.gpu_hashrate, data.gpu_usage, format_balance(data.estimated_earning)
+                    "CPU:\n  Hashrate: {} H/s\n  Usage: {:.0}%\nGPU:\n  Hashrate: {} H/s\n  Usage: {:.0}%\nEst. earning: {} tXTM/day",
+                    Formatter::new().with_decimals(2).with_separator("").format(data.cpu_hashrate),
```

Comment:
```suggestion
                    Formatter::new().with_decimals(0).with_separator("").format(data.cpu_hashrate),
```
I think 0 decimals is sufficient